ee/clickhouse/queries/funnels/funnel_persons.py

```diff
@@ -1,15 +1,31 @@
 from ee.clickhouse.queries.funnels.base import ClickhouseFunnelBase
-from ee.clickhouse.sql.funnels.funnel import FUNNEL_PERSONS_SQL
+from ee.clickhouse.sql.funnels.funnel import FUNNEL_PERSONS_BY_STEP_SQL
 from posthog.models import Person
 
 
 class ClickhouseFunnelPersons(ClickhouseFunnelBase):
```

Comment:
Another thought for the future / something I'll implement with ClickhouseFunnelUnordered + Persons / when this one gets merged in.

Since every funnel query will now take the form:
`ClickhouseFunnel` & `ClickhouseFunnelPersons`,
`ClickhouseFunnelUnordered` & `ClickhouseFunnelUnorderedPersons`,
`ClickhouseFunnelTrends` & `ClickhouseFunnelTrendsPersons`, (?)
`ClickhouseFunnelStrict` & `ClickhouseFunnelStrictPersons`,

I want to change how we do these, such that: `Persons` inherits from the `non Persons` bit, and the `non-Persons` class doesn't add the final outer aggregation in it's private function, so that the `Persons` query can reuse that.

Further, I think since at least `Strict` and `Unordered` funnels would use the same step logic as here, I'll probs convert that into `ClickhouseFunnelPersonsBase` and inherit from both to build this query.

In effect, all 3 would look the same. (not sure yet about trends, but 3 is pattern enough to abstract I think 😂 )

---

Any objections to this?

Reply:
No objections I think that makes the most sense